### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by jjw24**

### CODE REVIEW

**Comment by jjw24** [comment]
@taooceros I didn't see the resolve conflict, do we want to resolve or just close this for a new pr?

**Review by onesounds — APPROVED** [review]

**Review by onesounds — DISMISSED** [review]

File: Plugins/Flow.Launcher.Plugin.Explorer/Languages/ko.xaml [hunk 1]
```diff
@@ -1,112 +1,116 @@
...
+    <system:String x:Key="plugin_explorer_deletefilefoldersuccess_detail">Successfully deleted {0}</system:String>
+    <system:String x:Key="plugin_explorer_globalActionKeywordInvalid">Assigning the global action keyword could bring up too many results during search. Please choose a specific action keyword</system:String>
+    <system:String x:Key="plugin_explorer_quickaccess_globalActionKeywordInvalid">Quick Access can not be set to the global action keyword when enabled. Please choose a specific action keyword</system:String>
+    <system:String x:Key="plugin_explorer_windowsSearchServiceNotRunning">The required service for Windows Index Search does not appear to be running</system:String>
+    <system:String x:Key="plugin_explorer_windowsSearchServiceFix">To fix this, start the Windows Search service. Select here to remove this warning</system:String>
+    <system:String x:Key="plugin_explorer_alternative">The warning message has been switched off. As an alternative for searching files and folders, would you like to install Everything plugin?{0}{0}Select 'Yes' to install Everything plugin, or 'No' to return</system:String>
```
Comment (jjw24) [inline Plugins/Flow.Launcher.Plugin.Explorer/Languages/ko.xaml:16]: @onesounds is this correct?

File: Plugins/Flow.Launcher.Plugin.Explorer/Languages/ko.xaml [hunk 2]
```diff
@@ -1,112 +1,116 @@
...
+    <system:String x:Key="plugin_explorer_excludedfromindexsearch_msg">Excluded from Index Search</system:String>
+    <system:String x:Key="plugin_explorer_openindexingoptions">윈도우 인덱싱 옵션 열기</system:String>
+    <system:String x:Key="plugin_explorer_openindexingoptions_subtitle">Manage indexed files and folders</system:String>
+    <system:String x:Key="plugin_explorer_openindexingoptions_errormsg">윈도우 인덱싱 옵션 열기에 실패했습니다</system:String>
+    <system:String x:Key="plugin_explorer_add_to_quickaccess_title">Add to Quick Access</system:String>
+    <system:String x:Key="plugin_explorer_add_to_quickaccess_subtitle">Add current item to Quick Access</system:String>
+    <system:String x:Key="plugin_explorer_addfilefoldersuccess">성공적으로 추가되었습니다</system:String>
+    <system:String x:Key="plugin_explorer_addfilefoldersuccess_detail">Successfully added to Quick Access</system:String>
+    <system:String x:Key="plugin_explorer_removefilefoldersuccess">성공적으로 제거했습니다</system:String>
+    <system:String x:Key="plugin_explorer_removefilefoldersuccess_detail">Successfully removed from Quick Access</system:String>
```
Comment (jjw24) [inline Plugins/Flow.Launcher.Plugin.Explorer/Languages/ko.xaml:94]: @onesounds and this, some have reverted back to eng?

File: Plugins/Flow.Launcher.Plugin.Explorer/Languages/ko.xaml [hunk 3]
```diff
@@ -117,18 +121,18 @@
...
+    <system:String x:Key="flowlauncher_plugin_everything_installing_subtitle">Installing Everything service. Please wait...</system:String>
+    <system:String x:Key="flowlauncher_plugin_everything_installationsuccess_subtitle">Successfully installed Everything service</system:String>
+    <system:String x:Key="flowlauncher_plugin_everything_installationfailed_subtitle">Failed to automatically install Everything service. Please manually install it from https://www.voidtools.com</system:String>
+    <system:String x:Key="flowlauncher_plugin_everything_run_service">Click here to start it</system:String>
+    <system:String x:Key="flowlauncher_plugin_everything_installing_select">Unable to find an Everything installation, would you like to manually select a location?{0}{0}Click no and Everything will be automatically installed for you</system:String>
+    <system:String x:Key="flowlauncher_plugin_everything_enable_content_search">Do you want to enable content search for Everything?</system:String>
+    <system:String x:Key="flowlauncher_plugin_everything_enable_content_search_tips">It can be very slow without index (which is only supported in Everything v1.5+)</system:String>
```
Comment (jjw24) [inline Plugins/Flow.Launcher.Plugin.Explorer/Languages/ko.xaml:136]: This too

**Review by jjw24 — COMMENTED** [review]
see comments

**Pull request closed without merging**